src/deepsparse/transformers/pipelines/question_answering.py

```diff
@@ -147,6 +147,32 @@ def __init__(
 
         super().__init__(**kwargs)
 
+    def __call__(self, *args, **kwargs) -> BaseModel:
```

Comment:
does this call override add something besides unpacking `engine_inputs, postprocess_kwargs` this should already be supported by the base class right?

Reply:
The main difference this override adds is on line 162 where it calls the engine multiple times if you have more than one set of inputs. The unpacking is just something I simplified for this specific case.

Reply:
ah. nice, missed that. is that to support more spans? would prefer to instead add an overridable method in the base Pipeline `def engine_forward(self, inputs, **kwargs)` that just does the engine forward, and then override in the QA pipeline. that way if there's changes to `__call__` in the future they will be propagated to QA

Reply:
implementation looks great!